src/status_im/contexts/wallet/events/collectibles.cljs

```diff
@@ -24,9 +24,11 @@
 
 (defn displayable-collectible?
   [collectible]
-  (let [{:keys [image-url animation-url]} (:collectible-data collectible)]
+  (let [{{:keys [image-url animation-url]} :collectible-data
+         {collection-image-url :image-url} :collection-data} collectible]
     (or (not (string/blank? animation-url))
-        (not (string/blank? image-url)))))
+        (not (string/blank? image-url))
+        (not (string/blank? collection-image-url)))))
```

Comment:
Now `displayable-collectible?` is more flexible, it considers the collection images